src/plugins/intel_cpu/src/nodes/pooling.cpp

```diff
@@ -174,7 +174,20 @@ Pooling::Pooling(const std::shared_ptr<ov::Node>& op, const GraphContext::CPtr c
         }
     };
 
-    if (auto maxPoolOp_v8 = ov::as_type_ptr<const ov::op::v8::MaxPool>(op)) {
+    if (auto maxPoolOp_v14 = ov::as_type_ptr<const ov::op::v14::MaxPool>(op)) {
```

Comment:
Can here be used max pool base class to extract common attributes?
Then if required do cast for specific version for any specific attribute(s).

Reply:
Done, but for `MaxPool` only since `AvgPool` doesn't have a base class. I could create a GFI for it, could be useful in Core as well.